dominos/src/main/resources/application.properties

```diff
@@ -7,4 +7,10 @@ spring.datasource.password=mariopizza
 spring.jpa.hibernate.ddl-auto=update
 
 # Show SQL statements; good for debugging
-spring.jpa.show-sql=true
\ No newline at end of file
+spring.jpa.show-sql=false
+
+
+spring.jpa.properties.hibernate.generate_statistics = true
+spring.jpa.properties.hibernate.jdbc.batch_size = 30
```

Comment:
You played around with these settings last session, right? Did they make any significant change? If the impact is neglectable, then we can remove the setting. If it adds value, we can keep it

Reply:
I tried to get the import time to an acceptable level. With the current settings its not possible to do any import of 600 000 records unless you want to longer than 1 hour. The impact is enormous. And I already explained and provided documentation in teams before I made this pull request. If you remove these settings then it's not going to be able to do these amounts of records.

Reply:
Addendum:

The following options are required:

- spring.jpa.show-sql=false
  - If this is on, it has huge detrimental impact on performance.
- spring.jpa.properties.hibernate.generate_statistics = true
  - this is not needed, but it outputs after the entire saveAll request is done, a set of statistics that proove both timing and use of batching of queries. While in debugging/test this is essential to have.
-  spring.jpa.properties.hibernate.jdbc.batch_size = 30
  - When enabling batches its recommended not to go higher than 30.
-  spring.jpa.properties.hibernate.jdbc.batch_versioned_data = true
  - Recommended to use, but not yet tested when turned off. This is a todo.
- spring.jpa.properties.hibernate.order_inserts=true
  - Without this, I found enormous impact on many inserts. So it's necessary to have this.  

Having said this, I do not know if it is possible to configure these options for just only 1 part of the code. This is also a todo, then the settings might be limited to only the mdb part.

Reply:
Thanks for the clarification. The `spring.jpa.properties.hibernate.generate_statistics` can probably be removed as it currently clutters the log. We can enable it locally during development when we need it.

> Having said this, I do not know if it is possible to configure these options for just only 1 part of the code. This is also a todo, then the settings might be limited to only the mdb part.

Regarding this, I don't think this matters too much. I only import 40k rows, @mdevrees only 145 rows. Not sure how many @teuncortooms faces. Your import likely face the biggest performance bottleneck. I doubt these settings will negatively impact the rest, but we'll test it once.